### PR TITLE
{tools}[dummy/dummy] Aspera-CLI v3.7.2

### PR DESCRIPTION
**Pull request by ddkinnamon**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Aspera CLI install script includes tarball inline and installs to a
+# fixed location. Need to include custom extract command to pull
+# tarball out of install script and pipe to tar. %s in the extraction
+# command will be replaced by the source filename.
+sources = [
+    ('%(namelower)s-%(version)s%(versionsuffix)s-linux-64-release.sh',
+     "sed '1,/^__ARCHIVE_FOLLOWS__$/d' %s |" +
+     'tar -xzpo --strip-components 1 -f -')
```
Comment (boegel) [inline easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb]: urgh, what a mess, but I don't see a better option...

@ddkinnamon I would prefer having the extract command on a singe line though, it looks a bit awkward now with the `|` at the end...


```python
sources = [
    ('%(namelower)s-%(version)s%(versionsuffix)s-linux-64-release.sh',
     "sed '1,/^__ARCHIVE_FOLLOWS__$/d' %s | tar -xzpo --strip-components 1 -f -")
```

File: easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+]
+source_urls = ['http://download.asperasoft.com/download/sw/cli/%(version)s']
+checksums = ['02787ca46814fb9ae2de5c706461367e']
+
+sanity_check_paths = {
+    'files': ['product-info.mf'],
+    'dirs': ['bin', 'certs', 'docs', 'etc', 'share/man']
+}
+
+sanity_check_commands = ['ascp -h', 'man ascp -P cat']
```
Comment (boegel) [inline easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb]: @damianam the `man ascp -P cat` command seems to run for me, but exists with exit code `16` (even though there's no trace of an error), which makes the sanity check fail?

Comment (damianam) [inline easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb]: @boegel wrong tag?

Comment (boegel) [inline easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb]: sorry, was meant for @ddkinnamon

File: easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb (hunk 1 above)
Comment (ddkinnamon) [inline easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb]: @boegel: Will modify. I was trying not to exceed the sacred 80-character column width, but I agree that the single line is clearer.

Comment (boegel) [inline easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb]: @ddkinnamon We maintain a more flexible 120-character column limit. ;-)

**Comment by ddkinnamon** [comment]
Test report by @ddkinnamon
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
oakley01.osc.edu - Linux RHEL 6.7, Intel(R) Xeon(R) CPU           X5650  @ 2.67GHz, Python 2.7.13
See https://gist.github.com/64f6ff3ced7841418132441dca961e46 for a full test report.

File: easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb (hunk 2 above)
Comment (ddkinnamon) [inline easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb]: @boegel: I'm having trouble reproducing your `man` error. I just uploaded the test report from my system on which this returns with exit code 0. I'm running version 1.6f of `man`. Does it work for you without the `-P cat`?

Comment (boegel) [inline easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb]:
```
$ man --version
man 2.6.3
$ rpm -qa | grep ^man
man-db-2.6.3-9.el7.x86_64
```

works fine with just `man ascp` for me

Comment (ddkinnamon) [inline easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb]: @boegel: Was able to reproduce with `man` 2.6.3 on our RHEL 7 cluster. Changed to `man ascp` and eliminated error in test run.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2641.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/567cf4e5ea6e5cae241af02e61916d26 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2041.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cd6727922dc5b5cd51b5981dfe9aa72f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ddkinnamon!

**Pull request merged**